src/buttons/flat-button-label.jsx

```diff
@@ -61,19 +71,16 @@ const FlatButtonLabel = React.createClass({
       style,
     } = this.props;
 
-    const contextKeys = this.constructor.getRelevantContextKeys(this.state.muiTheme);
+    const {
+      prepareStyles,
+    } = this.state.muiTheme;
 
-    const mergedRootStyles = this.mergeStyles({
-      position: 'relative',
-      paddingLeft: contextKeys.spacingDesktopGutterLess,
-      paddingRight: contextKeys.spacingDesktopGutterLess,
-    }, style);
+    const styles = getStyles(this.props, this.state);
 
     return (
-      <span style={this.prepareStyles(mergedRootStyles)}>{label}</span>
+      <span style={prepareStyles(styles.root, style)}>{label}</span>
```

Comment:
from what I understand from the transformer chain that is `prepareStyles` it only takes 1 argument. so `style` will probably be ignored. I think you should `Object.assign({},...)` this too.